src/editors/containers/VideoEditor/components/VideoSettingsModal/components/TranscriptWidget/index.jsx

```diff
@@ -67,10 +106,10 @@ export const TranscriptWidget = ({
         {hasTranscripts ? (
 
           <Form.Group className="mt-4.5">
-            { Object.entries(transcripts).map(([language, value]) => (
-              <TranscriptListItem
+            { transcripts.map((language, index) => (
+              <Transcript
```

Comment:
Add some padding around this component. There is no spacing when the delete card appears and the transcript below it.

Reply:
Done! I added it in the delete card